### ISSUE TITLE
NPM install leads to vulnerabilities

### ISSUE THREAD

**Issue by VH-WRK** [bigcommerce/stencil-cli#507]
### Actual behavior
I've reinstalled stencil and am running version 1.19.0. When i run npm install i get the below vulnerabilities:
```
added 2899 packages from 2068 contributors and audited 19189 packages in 113.49s
found 414 vulnerabilities (58 low, 260 moderate, 94 high, 2 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
```
Should i be concerned?

Running:
node 8.1.0
Mac OS 10.12.6

**Comment by mikecalland** [comment]
I'm trying to use Stencil CLI for the first time and i am encountering this issue in terminal when installing Stencil CLI. 

I've installed NPM, installed the recommended version of Node from this page: https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil

I've ran the command: npm install -g @bigcommerce/stencil-cli

All of the items start with npm WARN deprecated. 

Any idea what is going on? i'm new to using command line and i can't find any help online.
